drivers/arm/gic/v2/gicv2_main.c

```diff
@@ -593,3 +593,12 @@ unsigned int gicv2_set_pmr(unsigned int mask)
 
 	return old_mask;
 }
+
```

Comment:
Just a comment regarding the commit message
suggest to change this line from
"This is helpful to modify the default configuration of the GIC SPI's, which..."
to
"This is helpful to modify the default configuration of the non-secure GIC SPI's, which..."

Reply:
Done